crates/typst-cli/src/compile.rs

```diff
@@ -3,11 +3,12 @@ use std::path::{Path, PathBuf};
 
 use chrono::{Datelike, Timelike};
 use codespan_reporting::diagnostic::{Diagnostic, Label};
-use codespan_reporting::term::{self, termcolor};
+use codespan_reporting::term;
 use ecow::{eco_format, EcoString};
 use parking_lot::RwLock;
-use rayon::iter::{IndexedParallelIterator, IntoParallelRefIterator, ParallelIterator};
-use termcolor::{ColorChoice, StandardStream};
+use rayon::prelude::{
```

Comment:
Why did the rayon import change?

Reply:
This appears to be a mistake that occured while rebasing.